services/federator/src/Federator/API.hs

```diff
@@ -45,7 +52,18 @@ data API route = API
         :> "users"
         :> Capture "fqu" (Qualified UserId)
         :> "prekeys"
-        :> Get '[JSON] PrekeyBundle
+        -- FUTUREWORK(federation):
+        -- this should return a version of PrekeyBundle with qualified UserId,
+        -- defined in wire-api-federation
+        :> Get '[JSON] PrekeyBundle,
+    _gapiJoinConversationById ::
```

Comment:
I'm not sure yet if the Federator needs to be _typed_ . in the sense that it's just a broker that has to decide where to fan out to right?  

The federator does very little at the moment. Do we actually need it?

Reply:
But lets worry about this after we have the entire feature implemented end to end